apps/video/js/view.js

```diff
@@ -497,10 +496,12 @@
 
     dom.playHead.classList.remove('active');
 
+    // We didn't move the play head while the user's finger was on it
+    // so now it may be in the wrong place.
+    updateSlider();
+
```

Comment:
yes, necessary. updateSlider() has the same `if (!dragging)` code, so we've got to call it again when the dragging ends.